### PR TITLE
Add Some Quotes about Language

### PR DESCRIPTION
**Pull request by rizastoday**
## ✅ New Quote Checklist
- [ ] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [ ] The quote(s) added are about **learning a foreign language**.
- [ ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [ ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

## ✅ New Language Checklist
- [ ] There is ❌ no assigned issue or pull request for this new language.
- [ ] ➕ Added a new JSON object in the `server/data/languages.json` file.
- [ ] Added the **voice** 🎤 attribute supported by ResponsiveVoice API.
- [ ] Added the **flag** 🏳 attribute in the format of the twemoji-awesome CSS syntax.
- [ ] Added the **romanization** attribute for appropriate languages.
- [ ] ➕ Created the `{language.json}` file inside the `server/data/words` folder containing the list of the 3000 words.
- [ ] Removed duplicates and noted 🗒 them in the Pull Request.
- [ ] Increment `currentLanguagesCount` in `client/src/store/index.js`
- [ ] Add new language in `README.md`

### CODE REVIEW

**Comment by rizastoday** [comment]
Hi @jayehernandez , this is my first contribution to an open-source project, please review it and give feedback if needed, I will be very enthusiastic,big thank you

**Review by jayehernandez — CHANGES_REQUESTED** [review]
Hey there @rizastoday! Congrats on your first contribution! Please add the quotes in alphabetical order, based on the author's first name. When that's done, you may see that some of the quotes you added are already duplicates, hope you can remove those.

Please request for review again, once done.

**Comment by rizastoday** [comment]
Hi there @jayehernandez , I've made a few changes such as sorting by author's first name and deleting the duplicate quotes that I previously added. Please re-review and let me know if there any of my changes go wrong. Thank you

File: server/data/quotes.json [hunk 1]
```diff
@@ -204,13 +212,17 @@
     "sentence": "Every language is a magnificent edifice which is built word by word, cemented by the rules of grammer and painted by colorful poetry."
   },
   {
-    "author": "Geoffrey Willans",
-    "sentence": "You can never understand one language until you understand at least two."
+    "author": "Gede Prama",
+    "sentence": "In the language of meditation, the past is past, the future is not here, now is the only gift available"
```
Comment (jayehernandez) [inline server/data/quotes.json:216]: Hope you can remove this quote as it is not about learning a foreign language.

File: server/data/quotes.json [hunk 2]
```diff
@@ -180,16 +188,16 @@
...
+    "sentence": "Speech is the great medium through which human co-operation is brought about."
   },
   {
     "author": "French Proverb",
     "sentence": "A man who knows two languages is worth two men."
   },
   {
-    "author": "Frederica de Laguna",
-    "sentence": "Speech is the great medium through which human co-operation is brought about."
+    "author": "French Proverb 2020",
```
Comment (jayehernandez) [inline server/data/quotes.json:199]: Remove the 2020 in the author name :)

File: server/data/quotes.json [hunk 3]
```diff
@@ -147,6 +151,10 @@
     "author": "Ellen Jovin",
     "sentence": "A new language is a hand held out to one’s neighbor, an opener of doors, a new way to see, a mental tickle, a road to unmediated communication with strangers in other lands, access to the world’s news, a gesture of peace."
   },
+  {
+    "author": "Eric Hoffer",
+    "sentence": "Leaders must be practical and realistic, but must speak visionary and idealistic language"
```
Comment (jayehernandez) [inline server/data/quotes.json:156]: Quote is also unrelated, hope you can remove this!

**Review by jayehernandez — CHANGES_REQUESTED** [review]

**Comment by jayehernandez** [comment]
Thanks for re-arranging them @rizastoday, just have a few more comments and PR is good to go. Hope you can resolve conflicts too.

**Comment by rizastoday** [comment]
I've made some changes regarding removing some unrelated quotes and I've tried to remove the conflict too. Hope you review it @jayehernandez  thank you

**Review by jayehernandez — APPROVED** [review]

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @rizastoday!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**